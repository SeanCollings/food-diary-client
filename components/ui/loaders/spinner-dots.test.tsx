import { render } from '@testing-library/react';
import { SpinnerDots } from './';

describe('<SpinnerDots>', () => {
  it('should render', () => {
    const { asFragment } = render(<SpinnerDots />);
    expect(asFragment()).toMatchSnapshot();
  });
});
