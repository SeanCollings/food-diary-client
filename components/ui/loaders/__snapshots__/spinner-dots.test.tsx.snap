// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`<SpinnerDots> should render 1`] = `
<DocumentFragment>
  <span
    class="sc-ceb7627d-0 wSPCm loader"
    color="#ffffff"
    size="45"
  />
</DocumentFragment>
`;
